modules/ROOT/pages/tutorial-initial-setup.adoc

```diff
@@ -0,0 +1,101 @@
+curl -O https://builds.coreos.fedoraproject.org/prod/streams/stable/builds/$RELEASE/x86_64/fedora-coreos-$RELEASE-qemu.x86_64.qcow2.xz.sig
+----
+
+Once the archive has been downloaded, make sure to verify its integrity by following the instructions available by clicking on the `Verify signature & SHA256` button. You will have to download the checksum file, the signature and Fedora GPG keys to verify your download:
+
+[source,bash]
+----
+curl https://getfedora.org/static/fedora.gpg | gpg --import
+gpg --verify fedora-coreos-$RELEASE-qemu.x86_64.qcow2.xz.sig fedora-coreos-$RELEASE-qemu.x86_64.qcow2.xz
+----
+
+Once you have verified the archive, you can extract it with:
+
+[source,bash]
+----
+unxz fedora-coreos-$RELEASE-qemu.x86_64.qcow2.xz
+----
+
+To make the tutorial simpler, you should rename this image to a shorter name:
+
+[source,bash]
+----
+mv fedora-coreos-$RELEASE-qemu.x86_64.qcow2 fedora-coreos.qcow2
+----
+
+== Installing `fcct` and `ignition-validate`
+
+To generate and validate Ignition configuration files, we need the following tools:
+
+- `fcct`: Fedora CoreOS Config Transpiler,
+- `ignition-validate`: Validate Ignition configuration files.
+
+If you are running on Fedora, you can install them via:
+
+[source,bash]
+----
+sudo dnf install -y fcct ignition-validate
+----
+
+You may also run them in a container by using either podman or Docker:
+
+[source,bash]
+----
+podman pull quay.io/coreos/fcct
+podman pull quay.io/coreos/ignition-validate
+----
+
+For example:
+
+[source,bash]
+----
+podman run --rm --interactive quay.io/coreos/ignition-validate - < myconfig.ign
+----
+
+Or you may directly download the latest https://github.com/coreos/fcct/releases[fcct] and https://github.com/coreos/ignition/releases[ignition-validate] releases from GitHub:
+
+[source,bash]
+----
+# fcct
+curl -O https://github.com/coreos/fcct/releases/download/latest/fcct-x86_64-unknown-linux-gnu
+curl -O https://github.com/coreos/fcct/releases/download/latest/fcct-x86_64-unknown-linux-gnu.asc
+gpg verify fcct-x86_64-unknown-linux-gnu.asc fcct-x86_64-unknown-linux-gnu
+mv fcct-x86_64-unknown-linux-gnu fcct
+
+# ignition-validate
+curl -O https://github.com/coreos/ignition/releases/download/latest/ignition-validate-x86_64-linux
+curl -O https://github.com/coreos/ignition/releases/download/latest/ignition-validate-x86_64-linux.asc
+gpg verify ignition-validate-x86_64-linux.asc ignition-validate-x86_64-linux
+mv ignition-validate-x86_64-linux ignition-validate
+----
+
+You may then setup aliases for `fcct` and `ignition-validate`:
+
+[source,bash]
+----
+alias fcct="${PWD}/fcct"
+alias ignition-validate="${PWD}/ignition-validate"
+----
+
+Or move those commands to a folder in your `$PATH`, for example:
+
+[source,bash]
+----
+mv fcct ignition-validate "${HOME}/.local/bin/"
+# Or
+mv fcct ignition-validate "${HOME}/bin"
+----
+
+You are now ready to proceed with the xref:tutorial-basic-provisioning.adoc[first tutorial].
```

Comment:
I wonder if we should either make each tutorial lead to the next (like this intro links to the First tutorial),

OR

We just say here that now you can proceed with executing one of the tutorials and link back to the tutorials page. This means we could add more tutorials in the future and not have to maintain a bunch of references.

Reply:
Adding link to the next Provisioning scenario sounds like a good idea.